test/acceptance/features/serviceClaim.feature

```diff
@@ -74,6 +74,8 @@ Feature: Service claim with label selector
         """
         Then On Primaza Cluster "primaza-main", the status of ServiceClaim "sc-test" is "Resolved"
         And On Primaza Cluster, "primaza-main", the secret "sc-test" has the key "type" with value "psqlserver"
+        And On Primaza Cluster "primaza-main", RegisteredService "primaza-rsdb" state will eventually move to "Claimed"
+        And On Primaza Cluster "primaza-main", ServiceCatalog "primaza-service-catalog" will not contain RegisteredService "primaza-rsdb"
```

Comment:
we could add here also a test for the scenario in which SED is not matching

Reply:
Agree, will give it a try, but same comment, would expect @baijum to pick up if I get stuck